### PR TITLE
feat: add alt tags and presentation roles

### PR DESCRIPTION
**Pull request by tilmx**
Fixes #60, #28

### CODE REVIEW

**Comment by marionebl** [comment]
Deploy preview processing.

Built with commit fe75022b7d7074ed4682bbe9b9ebd05ea56344e3

https://app.netlify.com/sites/determined-cray-3fd905/deploys/5a427b024c4b93025a55dd7e

File: lib/index.html [hunk 1]
```diff
@@ -27,16 +26,16 @@
 		<div class="wrapper left">
 			<div class="hero-links">
 				<a href="https://github.com/meetalva/alva" target="_blank" class="hero-github" title="Alva on Github">
-					<img src="images/github.svg" role="presentation">
+					<img src="images/github.svg" role="presentation" alt="">
```
Comment (lkuechler) [inline lib/index.html:29]: alt text could be `Github logo`

File: lib/index.html [hunk 2]
```diff
@@ -27,16 +26,16 @@
...
 				</a>
 				<a href="https://twitter.com/meetalva" target="_blank" title="Alva on Github">@meetalva on Twitter</a>
 			</div>
-			<h1><b>Meet Alva,</b><br> a radically new <u>digital design tool</u> built for cross-functional product teams.</h1>
+			<h1><strong>Meet Alva,</strong><br> a radically new <u>digital design tool</u> built for cross-functional product teams.</h1>
 			<p>Alva lets you design interactive products based on the same components your engineers are using for production websites. But, our mission goes way further. And guess what – we are entirely <a href="https://github.com/meetalva" target="_blank">open source</a>.</p>
 			<a id="download-btn" class="button js-download-button" href="https://github.com/meetalva/alva/releases/latest">Download Alva</a>
 			<a href="https://github.com/meetalva/alva" target="_blank" class="hero-github">
-				<img src="images/github.svg">
+				<img src="images/github.svg" alt="" role="presentation">
```
Comment (LKuechler) [inline lib/index.html]: same as above

File: lib/index.html [hunk 3]
```diff
@@ -74,23 +73,23 @@ <h2>Design and engineering, joined together</h2>
 			<section class="feature-tiles">
 				<div class="feature">
 					<div class="tile">
-						<img src="./images/wireframe.png" />
+						<img src="./images/wireframe.png" alt="Image of a wireframe" />
```
Comment (lkuechler) [inline lib/index.html]: Is the word "image" not a bit redundant here?

File: lib/index.html (hunk 1 above)
Comment (tilmx) [reply lib/index.html:29]: Yes – but directly next to it is a text saying "Alva on Github"?

File: lib/index.html (hunk 3 above)
Comment (tilmx) [reply lib/index.html]: That's right. Should we just use alt="" and role="presentation"?

File: lib/index.html [hunk 4]
```diff
@@ -105,7 +104,7 @@ <h2>Design and engineering, joined together</h2>
 			<h2>Cross-functional single source of truth</h2>
 			<p>Instead of creating a separate static component library for Sketch, Photoshop and all the other tools, you can start building your layouts and design with the very same components that your engineers use for the production website.</p>
 
-			<img src="images/components.png" class="source-image">
+			<img src="images/components.png" class="source-image" alt="" role="presentation">
```
Comment (LKuechler) [inline lib/index.html]: Could be `List of example components`

File: lib/index.html [hunk 5]
```diff
@@ -138,14 +137,14 @@ <h4>Interactive</h4>
 			</div>
 
 			<div class="features-additional">
-				<img src="images/icon-customcode.png" aria-hidden="true">
+				<img src="images/icon-customcode.png" aria-hidden="true" alt="" role="presentation">
```
Comment (lkuechler) [inline lib/index.html:140]: I don't have a good alt text for this one and its already `aria-hidden` but we also should not have empty alt texts. If we don't have a text for them and they are hidden then there should probably not be a alt text at all

Comment (tilmx) [reply lib/index.html:140]: A11y says something different here: 
"Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page"
https://a11yproject.com/posts/alt-text/

Comment (lkuechler) [reply lib/index.html:140]: Ok i just read about it again and webaim.org for example says that we should leave it empty.
Strange but we can do it this way

File: src/style.less [hunk 6]
```diff
@@ -299,7 +299,7 @@ input[type=email] {
             .font-medium;
         }
 
-		u {
+		strong {
```
Comment (lkuechler) [inline src/style.less:302]: We should style classes and not tags but as i see this is a bit late for the website project ;)

File: lib/index.html (hunk 3 above)
Comment (lkuechler) [reply lib/index.html]: I think the whole feature tiles section will not make any sense to someone that is visually impaired.
You will just hear:
```
"... and replace all the placeholder components."
"Wireframe"
"Design"
"Design & Code"
```

I think the complete section is just visual presentation

File: src/style.less (hunk 6 above)
Comment (tilmx) [reply src/style.less:302]: I guess we should refactor some parts of the websites next year, that could be part of it...

Comment (lkuechler) [reply src/style.less:302]: maybe we could also create a lsg for the website. Then we could use Alva to try stuff out :)

File: lib/index.html (hunk 1 above)
Comment (tilmx) [reply lib/index.html:29]: added an empty alt="" tag. a screenreader saying "Github logo; Alva on Github" doesn’t make so much sense, does it?

File: src/style.less (hunk 6 above)
Comment (tilmx) [reply src/style.less:302]: We should think about that... The downside would be, that changes might get more complicated, but let’s discuss that in #62

File: lib/index.html (hunk 1 above)
Comment (lkuechler) [reply lib/index.html:29]: sounds good

**Review by lkuechler — APPROVED** [review]

**Comment by tilmx** [comment]
Fixes #24

**Pull request merged**